app/controllers/concerns/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] Kindly delete this controller. It is not needed. The one mentioned above is the correct controller.